src/_main.ts

```diff
@@ -62,12 +62,16 @@ export default class Main {
     Main.logger.info(`Started bots in ${time} ms.`);
   }
 
-  /** Starts the updater. */
-  public static async startUpdater(): Promise<void> {
-    if (Updater.getUpdater().enabled) {
-      Updater.getUpdater().start();
-      Updater.logger.info('Started updater.');
-    }
+  /** Starts the updaters. */
+  public static async startUpdaters(): Promise<void> {
+    const updaters = Updater.getUpdaters();
+
+    updaters.forEach((updater) => {
+      if (updater.enabled) {
+        updater.start();
+        updater.logger.info('Started updater.');
```

Comment:
This would just log `started updater` multiple times right? Can we log which updater started?

Reply:
It does with the updater label. It looks like this: 
```
[1] 2020-09-02 10:05:27 [Updater (reddit)]       info:   Started updater.
[1] 2020-09-02 10:05:27 [Updater (steam)]        info:   Started updater.
[1] 2020-09-02 10:05:27 [Updater (rss)]  info:   Started updater.
[1] 2020-09-02 10:05:27 [Updater (dota)]         info:   Started updater.
```